bundles/org.openhab.core.io.rest.sitemap/src/main/java/org/openhab/core/io/rest/sitemap/internal/SitemapResource.java

```diff
@@ -614,18 +591,18 @@ private PageDTO createPageBean(String sitemapName, @Nullable String title, @Null
         }
         bean.widgetId = widgetId;
         bean.icon = itemUIRegistry.getCategory(widget);
-        bean.staticIcon = widget.getStaticIcon() != null || !widget.getIconRules().isEmpty();
+        bean.staticIcon = widget.isStaticIcon() || !widget.getIconRules().isEmpty();
```

Comment:
bean.staticIcon = (widget.getIcon() != null && widget.isStaticIcon()) || !widget.getIconRules().isEmpty();

Reply:
I am not sure this is necessary.